### PR TITLE
3 cleanup commits

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -2,17 +2,92 @@ include: package:pedantic/analysis_options.yaml
 analyzer:
   strong-mode:
     implicit-casts: false
-  errors:
-    todo: ignore
```
Comment (natebosch) [inline analysis_options.yaml]: Why remove this?

Comment (kevmoo) [reply analysis_options.yaml]: Much noise. We're running travis w/ `--fatal-hints`

Comment (natebosch) [reply analysis_options.yaml]: This doesn't add any noise... It doesn't impact travis at all.

Comment (kevmoo) [reply analysis_options.yaml]: Noise to this file. Why these? Why not others? We fail CI for any items, so flagging some as errors vs others doesn't add much – AFAIC

**Pull request merged**